### PR TITLE
DOC-529: Improve wording of occupancy feature

### PR DESCRIPTION
**Pull request by m-hulbert**
> _NOTE TO REVIEWERS_ - please **do not review PRs in the `DRAFT` state**, as the PR may change substantially before it is ready to review. Thanks.

## Description

This PR addresses the feedback received in this [PR](https://github.com/ably/docs/pull/1209) to improve the wording and accuracy of the occupancy topic.

## Review

To review this PR, view the [occupancy page]().

### CODE REVIEW

**Review by paddybyers — APPROVED** [review]
LGTM, thanks

File: content/core-features/occupancy.textile [hunk 1]
```diff
@@ -2,16 +2,16 @@
...
+Occupancy differs from "presence":/core-features/presence in that presence enables a client to identify themselves to other clients subscribed to presence events on a channel, as well as to update their member data for subscribers to view. Presence events are emitted when a client enters or leaves a channel or when they update their member data. Occupancy provides a count of the clients attached to a channel, broken down into specific metric categories. Occupancy data can be viewed at a specific point in time when using the channel status API, or emitted as events when metrics change using a @channel.occupancy@ rule source or inband channel occupancy events.
```
Comment (marklewin) [inline content/core-features/occupancy.textile:17]: This all seems a bit wordy. Perhaps something like the following?

The occupancy and presence features differ in the following ways:

**Occupancy**: Provides information about the _number of members in the channel_. This includes a count of the number of clients attached to the channel, the number of publishers and subscribers, and how many clients subscribe to presence events.

**Presence**: Provides information about the _activity of individual channel members_. Clients attached to a channel announce their state to other clients in the channel via presence events. For example, when entering a channel, leaving a channel, or updating their member data. Other clients can subscribe to these events.

File: content/core-features/presence.textile [hunk 2]
```diff
@@ -11,4 +11,4 @@ index: 27
...
\ No newline at end of file
+"Occupancy":/core-features/occupancy differs from presence in that presence enables a client to identify themselves to other clients subscribed to presence events on a channel, as well as to update their member data for subscribers to view. Presence events are emitted when a client enters or leaves a channel or when they update their member data. Occupancy provides a count of the clients attached to a channel, broken down into specific metric categories. Occupancy data can be viewed at a specific point in time when using the channel status API, or emitted as events when metrics change using a @channel.occupancy@ rule source or inband channel occupancy.
```
Comment (marklewin) [inline content/core-features/presence.textile:14]: See comment in `occupancy.textile`.

File: content/core-features/occupancy.textile [hunk 3]
```diff
@@ -2,16 +2,16 @@
...
+The metrics returned by the occupancy feature include the number of clients attached to a channel, the number of those authorized to subscribe to messages on the channel and the number of those clients authorized to publish to the channel. The occupancy metrics also include the number of connections authorized to subscribe to presence events, the number authorized to publish presence events and the number of "presence":/core-features/presence members currently present on the channel.
 
-Channel occupancy can be queried in a number of ways. Occupancy can be viewed as part of "channel enumeration":/rest-api#enumeration-rest, configured as an "integration rule source":/general/events#sources or as "inband channel occupancy events":/realtime/inband-occupancy. 
+Channel occupancy can be queried in a number of ways. Occupancy can be viewed as part of the "channel status API":/rest/channel-status, when configured as a "@channel.occupancy@ rule source":/general/events#sources or as "inband channel occupancy events":/realtime/inband-occupancy.
```
[the remark this answers is not in the document]
Comment (m-hulbert) [reply content/core-features/occupancy.textile]: I've rewritten this paragraph to read more clearly. Please let me know if it still doesn't!

File: content/core-features/presence.textile (hunk 2 above)
Comment (m-hulbert) [reply content/core-features/presence.textile]: Applied the same change as made to `occupany.textile`.
